joplin/templates/wagtailadmin/pages/listing/_table_headers_explore.html

```diff
@@ -29,7 +29,7 @@
     {% endif %}
     <th class="title">
         {% if sortable %}
-            <a href="{% url 'wagtailadmin_explore' parent_page.id %}?ordering={% if ordering == 'title' %}-{% endif %}title" class="icon icon-arrow-{% if ordering == 'title' %}down-after{% elif ordering == '-title' %}up-after{% else %}down-after{% endif %} {% if ordering == 'title' or ordering == '-title' %}teal{% endif %}">
+            <a href="{% url 'wagtailadmin_explore' 3 %}?ordering={% if ordering == 'title' %}-{% endif %}title" class="icon icon-arrow-{% if ordering == 'title' %}down-after{% elif ordering == '-title' %}up-after{% else %}down-after{% endif %} {% if ordering == 'title' or ordering == '-title' %}teal{% endif %}">
```

Comment:
Might be worth leaving a note here/linking to where we talk about hard-coding?

Reply:
Is there a way to do this without hardcoding the page_id?

Reply:
the issue I found with using a non-hardcoded id is that we don't always have a parent_page in here, but we still want to have the explore button work.

We currently have every page as a child of "Home". We could have some logic to say "if we have a parent page, use that ID, if not default to 3" but if we end up with a parent_page other than "Home" a bunch of other stuff would break anyways.